src/component/LayoutPlatform.js

```diff
@@ -145,46 +145,40 @@ const Page = ({ children, section, className, pageTitle, pageTitleDate, PageTitl
           <div className="container">
             <div className="col-wrapper one">
               <div className="coll">
-                <h2>Support</h2>
+                <h2>Community Supported</h2>
                 <p>
-                  The Wazo Platform project is supported by its community. Use the various channels below to reach out
-                  to us.
+                  Wazo Platform is supported by its community. Use our various channels to reach out.
                 </p>
+
+                <ul>
+                  <li>
+                    <a href="https://mm.wazo.community/wazo-platform/">Mattermost</a>
+                  </li>
+                  <li>
+                    <a href="https://wazo-platform.discourse.group">Forum</a>
+                  </li>
+                  <li>
+                    <a href="https://wazo-dev.atlassian.net/">Bug tracking (JIRA)</a>
+                  </li>
+                </ul>
               </div>
               <div className="coll">
                 <div>
                   <h2>Publications</h2>
                   <p>
-                    Video @ Kamailio World 2019: <a href="https://youtu.be/d1hOR27r4uY?t=2642">Dangerous Demos</a>
+                    Video @ Kamailio World: <a href="https://youtu.be/d1hOR27r4uY?t=2642">Dangerous Demos</a>
                   </p>
                   <p>
-                    Video @ the Telecom Application Development Summit 2018:{' '}
+                    Video @ the Telecom Application Development Summit:{' '}
                     <a href="https://youtu.be/JxKlxPDDnsk">What can you do with Wazo?</a>
                   </p>
                   <p>
-                    Video @ the Telecom Application Development Summit 2017:{' '}
+                    Video @ the Telecom Application Development Summit:{' '}
```

Comment:
Why remove the years?

Reply:
Sine we don't have recent publications, old date make feel like the project is unmaintained. Date don't add anything to the «publication/title». They will see the date on youtube anyway :)